Unnamed Roguelike Game/Ability.gd

```diff
@@ -0,0 +1,12 @@
+extends Label
```

Comment:
Adding an underscore to delta tells Godot that we mean for this variable to be unused. The _process function in Godot, like the other built-in GDScript functions, requires that you have access to the delta variable. If you don't use this variable, the compiler complains saying that you have unused variables in your code. Since I can't remove delta (as it is built-in with the function), I must instead use _delta to prevent Godot from complaining. It doesn't change anything functionally, and if in the future I want to use delta, I can by just removing the underscore.